services/Facebook.json

```diff
@@ -23,6 +23,47 @@
         "removeHelpButtons",
         "cleanUrls"
       ]
+    },
+    "devPaymentsTos": {
+      "location": "https://developers.facebook.com/policy/credits",
+      "contentSelector": "._4bl9",
+      "filters": [
+        "removeHelpButtons",
+        "cleanUrls"
+      ]
+    },
+    "communityPaymentsTos": {
+      "location": "https://www.facebook.com/payments_terms",
+      "contentSelector": ".prl",
+      "filters": [
+        "removeHelpButtons",
+        "cleanUrls"
+      ]
```

Comment:
I get an illustration image that has a tracker timestamp in it at the top of the page, and that should probably be filtered: `![](https://scontent.fcdg1-1.fna.fbcdn.net/v/t39.2365-6/851580_592653390766020_788624936_n.png?_nc_cat=109&_nc_sid=ad8a9d&_nc_ohc=-T-hXqyA5TIAX-cEiwN&_nc_ht=scontent.fcdg1-1.fna&oh=c902ef234dbfae5a4c7b86d145a60a16&oe=5F508440)`.